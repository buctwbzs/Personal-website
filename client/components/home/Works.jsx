import React, { Component } from 'react'
import styled from 'styled-components'
import crm from '../../public/image/CRM.png'
import dashbord from '../../public/image/dashbord.png'
import cmentor from '../../public/image/c-mentor.png'
import cmentee from '../../public/image/c-mentee.png'


const CartoonBox = styled.section`
  display: flex;
  flex-direction: column;
  width: 100%;
  .title{
    width: 100%;
    text-align:center;
    font-size: 2rem;
  }
  .works{
    display: flex;
    justify-content: space-between;
    flex-wrap: wrap;
    margin-bottom: 1rem;
    .wrap{
      margin: 0 auto 1rem auto;
    }
    img{
      display: block;
      max-width: 100%;
    }
  }
`

class Cartoon extends Component {
  render() {
    return (
      <CartoonBox>
        <p className="title">Works</p>
        <div className="works">
          {[crm, dashbord, cmentee, cmentor].map((v, i) => (
            <div className="wrap">
              <img src={v}
                alt={`work-${i}`}
                key={i}
              />
            </div>
          ))}
        </div>
      </CartoonBox>
    )
  }
}



export default Cartoon