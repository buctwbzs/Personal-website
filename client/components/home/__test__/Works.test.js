import React from 'react'
import renderer from 'react-test-renderer'
import Works from '../Works'

it('Works renders correctly', () => {
  const tree = renderer.create(<Works />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})