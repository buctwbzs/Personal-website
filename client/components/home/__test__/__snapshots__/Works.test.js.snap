// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works renders correctly 1`] = `
<section
  className="Works__CartoonBox-s1mg5kcs-0 dLdsgj"
>
  <p
    className="title"
  >
    Works
  </p>
  <div
    className="works"
  >
    <div
      className="wrap"
    >
      <img
        alt="work-crm.png"
        src="http://www.buctwbzs.com/static/images/personal-website/crm.png"
      />
    </div>
    <div
      className="wrap"
    >
      <img
        alt="work-dashbord.png"
        src="http://www.buctwbzs.com/static/images/personal-website/dashbord.png"
      />
    </div>
    <div
      className="wrap"
    >
      <img
        alt="work-c-mentee.png"
        src="http://www.buctwbzs.com/static/images/personal-website/c-mentee.png"
      />
    </div>
    <div
      className="wrap"
    >
      <img
        alt="work-c-mentor.png"
        src="http://www.buctwbzs.com/static/images/personal-website/c-mentor.png"
      />
    </div>
  </div>
</section>
`;
